src/sentry/integrations/slack/message_builder/base/block.py

```diff
@@ -52,6 +52,8 @@ def get_tags_block(tags) -> SlackBlock:
         for tag in tags:
             title = tag["title"]
             value = tag["value"]
+            # remove backticks from value, otherwise it will break the markdown
+            value = value.replace("`", "")
```

Comment:
what if it's a code block?

Reply:
the tag value is a code block? it will end up being a code block again because we put it inside backticks

Reply:
oh you mean multiple backticks? doesn't this remove them all

Reply:
if it ends up being a code block again then that's great

Reply:
also do we want to only replace them in the start and end? or all in general?

Reply:
this replaces them all, if there's any in the middle it will still break the markdown code block

Reply:
code block is triple, code text is single

Reply:
we just want code text... it won't preserve the code block

Reply:
but also who is sending code blocks as tags